api/src/services/comments.ts

```diff
@@ -147,18 +148,23 @@ export class CommentsService extends ItemsService {
 			const userID = mention.substring(1);
 
 			const user = await this.usersService.readOne(userID, {
-				fields: ['id', 'first_name', 'last_name', 'email', 'role.id', 'role.admin_access', 'role.app_access'],
+				fields: ['id', 'first_name', 'last_name', 'email', 'role.id'],
 			});
 
 			const accountability: Accountability = {
 				user: userID,
 				role: user['role']?.id ?? null,
-				admin: user['role']?.admin_access ?? null,
-				app: user['role']?.app_access ?? null,
+				admin: false,
+				app: false,
 				roles: await fetchRolesTree(user['role']?.id, this.knex),
 				ip: null,
 			};
 
+			const userGlobalAccess = await fetchGlobalAccessForUser(accountability, this.knex);
```

Comment:
@hanneskuettner Should this be `fetchGlobalAccess` instead? 🤔 Would it work with public user?

Reply:
You are correct, it should be `fetchGlobalAccess`, so the access from the users roles are considered as well.

Reply:
Thank you! Resolved via https://github.com/directus/directus/pull/23972/commits/7524de973c39f797da5ad7902e346089400023ad